tests/Refinery/String/MakeClickableTest.php

```diff
@@ -0,0 +1,53 @@
+<?php declare(strict_types=1);
+
+namespace ILIAS\Tests\Refinery\String;
+
+use PHPUnit\Framework\TestCase;
+use ILIAS\Refinery\String\MakeClickable;
+
+class MakeClickableTest extends TestCase
+{
+    public function testConstruct() : void
+    {
+        new MakeClickable();
+        $this->assertTrue(true);
+    }
+
+    public function testTransformFailure() : void
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $clickable = new MakeClickable();
+
+        $clickable->transform(3);
+    }
+
+    /**
+     * @dataProvider provideInput
+     */
+    public function testTransformSuccess(string $expected, string $input) : void
+    {
+        $clickable = new MakeClickable();
+
+        $this->assertEquals($expected, $clickable->transform($input));
+    }
+
+    public function provideInput() : array
+    {
+        return [
```

Comment:
I'd prefer some `Lorem Ipsum` texts instead of random (cat ran over the keyboard) characters :).